content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Manual rotation of CA certificates
+content_template: templates/task
+reviewers:
+- 
+---
+
+{{% capture overview %}}
+
+This page shows how to manually rotate certificate authority (CA) certificates.
+
+{{% /capture %}}
+
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+For more information about authentication in Kubernetes, see "[Authenticating](/docs/reference/access-authn-authz/authentication)".
+For more information about best practices for CA certificates, see "[Single root CA](docs/setup/best-practices/certificates/#single-root-ca)".
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+There are two ways to rotate or replace CA certificates.
+
+## Replace the cluster (recommended)
+
+This is the safest approach to rotate or replace the CA certificates.
+
+1. Create a new cluster with new CA certificates.
+
+2. Migrate the workloads to the new cluster.
+
+3. Remove or destroy the old cluster.
+
+## Rotate the CA certificates manually
+
+{{< caution >}}
+Kubeadm does not directly support rotating or replacement of CA certificates. Make sure to back up your certificate directory along with confs and any other necessary files.
+
+Manually rotating the CA works by timing when a Kubernetes cluster knows about new certificates. Do not restart any pods or update any component in the cluster while manually rotating the CA.
+
+This approach requires a considerable downtime of your cluster.
+
+{{< /caution >}}
+
+1. Back up your `/etc/kubernetes` directory, referred to as the *Kubernetes conf directory* and `/etc/kubernetes/pki`, referred to as the *Kubernetes certificates directory*.
+
+
+1. Distribute new CA certificates to all your control plane nodes in the kubernetes certificates directory.
+
+
+    Copy the CA certificate file or CA bundle (whichever you're using) on all nodes needed by the kubelet and the control plane component's `--client-ca-file` flag in the manifest files. 
+
+    {{< note >}}
+
+    Control plane components run as static pod manifests, managed by the kubelet. Components are not disrupted by in-place updates of CA certificates until someone kills or restarts an already running pod.
+
+    {{< /note >}}
+
+1. Get new certificates and private keys signed from the new CA for `apiserver.crt`, `apiserver-kubelet-client.crt`, and `front-proxy-client.crt` and replace them in the Kubernetes certificates directory.
+
+    To generate certificates and private keys for your cluster using openssl, see "[openssl](/docs/concepts/cluster-administration/certificates/#openssl)". 
+    
+    You can also use [cfssl](/docs/concepts/cluster-administration/certificates/#cfssl).
+
+
+1. Update the config map `cluster-info` in the kube-public namespace.
+
+    ```shell
+    base64_encoded_ca=$(/usr/bin/base64 /etc/kubernetes/pki/ca.crt)
```

Comment:
nit (code style):
```suggestion
    base64_encoded_ca="$(base64 /etc/kubernetes/pki/ca.crt)"
```
What do you think?

Reply:
I have a habit of using absolute path, it will anyways work on shell without /usr/bin, so I removed it and committed the changes.